breakers.gemspec

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'breakers/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'breakers'
+  spec.version       = Breakers::VERSION
+  spec.authors       = ['Aubrey Holland']
+  spec.email         = ['aubrey@adhocteam.us']
+
+  spec.summary       = 'Handle outages to backend systems with a Faraday middleware'
+  spec.description   = 'This is a Faraday middleware that detects backend outages and reacts to them'
+  spec.homepage      = ''
+  spec.license       = 'MIT'
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  if spec.respond_to?(:metadata)
+    spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+  else
+    raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
+  end
+
+  spec.files = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'faraday'
+  spec.add_dependency 'multi_json'
+  spec.add_dependency 'sinatra'
```

Comment:
It's for the dashboard, which is written as a mountable sinatra app. I can't think of a way to do that or something similar without adding dependencies. Maybe that means that the dashboard ought to be split out into its own gem? Really it's only required if you're making use of the dash.